docs/topics/settings.rst

```diff
@@ -604,6 +604,21 @@ If you want to disable it set to 0.
 
     This feature needs Twisted >= 11.1.
 
+.. setting:: DOWNLOAD_ALLOW_DATALOSS
+
+DOWNLOAD_ALLOW_DATALOSS
```

Comment:
Some ideas on how to avoid ambiguous "data loss" name:

* `DOWNLOAD_VALIDATE_CONTENT_LENGTH`
* `DOWNLOAD_DROP_PARTIAL`